packages/wdio-spec-reporter/src/index.ts

```diff
@@ -27,12 +27,17 @@ export default class SpecReporter extends WDIOReporter {
         failed: '✖'
     }
 
+    private _sauceLabsSharableLinks:boolean = true
```

Comment:
It will be automatically a boolean if you have a default value like this
```suggestion
    private _sauceLabsSharableLinks = true
```